recipes/spirv-tools/all/dependencies/dependencies-1.3.204.0.yml

```diff
@@ -0,0 +1 @@
+spirv-headers: "1.3.204.0"
```

Comment:
Seems a bit much to have 10 files for 10 lines 🤔 was there a technical reason for moving these out of the recipe?

Reply:
Avoid to change conanfile (and thus RREV of all spirv-tools versions) when you have to hardcode a specific dependency version for a given spirv-tools version.